indico/modules/events/registration/controllers/management/regforms.py

```diff
@@ -17,6 +17,7 @@
 from __future__ import unicode_literals
 
 from flask import redirect, flash, session
+from indico.core import signals
```

Comment:
this should be in the next line (i.e. an empty line after the flask import) - we group imports like this:

```
__future__

imports from the stdlib

imports from thirdparty libs (like flask)

imports from indico itself
```